components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -82,32 +82,23 @@ type Props = {
 };
 
 type State = {
-
+    show: boolean;
 };
 
 export default class SidebarChannel extends React.PureComponent<Props, State> {
     static defaultProps = {
         isDraggable: true,
     }
 
-    isCollapsed = (props: Props) => {
-        return props.isCategoryDragged || (props.isCategoryCollapsed && !this.props.isUnread && !props.isCurrentChannel);
-    }
-
-    componentDidUpdate(prevProps: Props) {
```

Comment:
as explained in https://github.com/mattermost/mattermost-webapp/pull/9732/files#r798465348 animating class has to be present always